extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { JupyterBookSection, IJupyterBookSectionV1, IJupyterBookSectionV2 } from '../contracts/content';
+import { BookVersion } from './bookModel';
+
+export class BookVersionHandler {
+
+	constructor() { }
+
+	public convertFrom(version: string, tableOfContents: JupyterBookSection): JupyterBookSection {
+		if (version === BookVersion.v1) {
+			return Object.assign(tableOfContents, {
```

Comment:
Why are you doing Object.assign here if you're setting all the fields manually, anyway? Same for the assign call below